test/Microsoft.AspNetCore.Razor.Tools.Test/ServerCommandTest.cs

```diff
@@ -47,15 +47,44 @@ public void WritePidFile_WorksAsExpected()
             }
             finally
             {
-                // Delete the file in case the test fails.
-                if (File.Exists(path))
+                // Cleanup after the test.
+                if (Directory.Exists(directoryPath))
                 {
-                    File.Delete(path);
+                    Directory.Delete(directoryPath, recursive: true);
                 }
             }
         }
 
-        private ServerCommand GetServerCommand(string pipeName)
+        [Fact]
+        public void GetPidFilePath_ReturnsCorrectDefaultPath()
+        {
+            // Arrange
+            var homeEnvVariable = PlatformInformation.IsWindows ? "USERPROFILE" : "HOME";
+            var expectedPath = Path.Combine(Environment.GetEnvironmentVariable(homeEnvVariable), ".dotnet", "pids", "build");
```

Comment:
Use a test function rather than do this especially since you control the result of `getEnv` now